packages/vue-docgen-api/tests/components/jsfile/button-js.test.ts

```diff
@@ -5,12 +5,20 @@ import { parse } from '../../../src/main'
 const button = path.join(__dirname, './MyButton.js')
 let docButton: ComponentDoc
 describe('tests button with pug', () => {
-	beforeEach(async done => {
+	beforeAll(async done => {
 		docButton = await parse(button, { nameFilter: 'Button' })
 		done()
 	})
 
 	it('should extract the export name', () => {
 		expect(docButton.exportName).toBe('Button')
 	})
+
+	it('should extract a component description', () => {
+		expect(docButton.description && docButton.description.length > 0).toEqual(true);
+	})
+
+	it('should have the correct content in the extracted definition', () => {
+		expect(docButton.description!.trim()).toBe('This is a button that represents a javascript only component, not a vue SFC')
```

Comment:
```suggestion
		expect(docButton.description).toBe('This is a button that represents a javascript only component, not a vue SFC')
```

I was hoping you would not need this trim

Reply:
@elevatebart I just thought it might increase fault tolerance, but if you think it can go, no issues here.